libs/datamodel/core/src/validator/standardise.rs

```diff
@@ -468,7 +469,11 @@ impl Standardiser {
                         let ds_field = dml::DataSourceField {
                             name: db_name.clone(),
                             field_type: *scalar_type,
-                            arity: field.arity,
+                            arity: match field.arity {
+                                // FIXME: superior hack. Talk to Marcus. This is a workaround for the behavior in row.rs for trait `ToSqlRow`
+                                FieldArity::List => FieldArity::Optional,
```

Comment:
There's special handling of `FieldArity::List` and a scalar type identifier in `row.rs`. I could not fix it in the source. So i fixed this here for  now.

Reply:
It's probably even right to fix it here. This code computes the `DataSourceField`s for a relation field. We used to compute `FieldArity::List` for list relation fields. So the sql connector assumed that the underlying column is a list. I am unsure whether it should be `::Optional` or `::List` though.